services/MerchStoreService.ts

```diff
@@ -214,6 +221,15 @@ export default class MerchStoreService {
         throw new NotFoundError(`Missing: ${missingItems}`);
       }
 
+      // Checks that hidden items were not ordered
+      const hiddenItem = Array.from(itemOptions.values())
+        .filter((o) => o.item.hidden)
+        .map((o) => o.uuid);
+
```

Comment:
nit: rename to `hiddenItems`, since there could be more than 1 hidden items

Reply:
Wait I just realized, doesn't the code block above already check if hidden items were ordered? It seems to be filtering only non-hidden items and returning the items that weren't non-hidden (i.e. were hidden). Referring to this block: https://github.com/acmucsd/membership-portal/blob/4a9d27d7f90f90d953d8d0111c452f27d1c03dff/services/MerchStoreService.ts#L208

Reply:
That code block only runs if the query doesn't find all options in the originalOrder so it doesn't actually run in this situation

Reply:
My understanding is that it warns the user of getting non-existent items